R/state.R

```diff
@@ -0,0 +1,29 @@
+check_pre_release_state <- function(which, force = FALSE) {
```

Comment:
Let's do one function that returns the current state:

- pre-release (the only state with 4 version number components)
- releasing, not ready
- ready to release
- submitted, waiting
- accepted